src/renderer/store/editor.js

```diff
@@ -66,10 +67,13 @@ const mutations = {
   },
   SET_USER_PREFERENCE (state, preference) {
     Object.keys(preference).forEach(key => {
-      if (preference[key]) {
+      if (typeof preference[key] !== 'undefined' && typeof state[key] !== 'undefined') {
         state[key] = preference[key]
       }
     })
+
+    const { preferLooseListItem } = state
```

Comment:
I think maybe you can put this `event emit` in `action: ASK_FOR_USER_PREFERENCE `.

Another thing, You need judge weather `preferLooseListItem` is changed before emit `update-prefer-loose-list-item`.

codes maybe like this:

```javascript
const { preferLooseListItem } = preferences
if (preferLooseListItem) {
  // bus.$emit('update-prefer-loose-list-item', preferLooseListItem)
}
```